Add avatar for login button in Header.vue

hoseinmirian opened the pull request.

hoseinmirian committed:
Autogenerated documentation files

hoseinmirian committed:
Add avatar for login button in Header.vue

terazus commented on src/components/Navigation/Header.vue:
```diff
@@ -71,9 +77,13 @@
             :small="$vuetify.breakpoint.mdAndDown"
             :x-large="$vuetify.breakpoint.xlOnly"
             class="mr-1 mt-sm-1 teal darken-2"
-            to="/accounts/profile"
```
Why did you remove this line? the link to the profile page doesn't work anymore once you are logged in.

hoseinmirian replied:
its been removed accidentally

terazus commented on src/components/Navigation/Header.vue:
```diff
@@ -71,9 +77,13 @@
             :small="$vuetify.breakpoint.mdAndDown"
             :x-large="$vuetify.breakpoint.xlOnly"
             class="mr-1 mt-sm-1 teal darken-2"
```
```suggestion
            class="mr-1 mt-sm-1 teal darken-2 pl-2"
```
Reduce padding left before the icon to harmonize the button

hoseinmirian replied:
added

hoseinmirian committed:
Fix redirection bug

hoseinmirian committed:
Delete doc

hoseinmirian committed:
Update doc

hoseinmirian committed:
Autogenerated documentation files

hoseinmirian committed:
Delete doc

hoseinmirian committed:
Autogenerated documentation files

hoseinmirian commented:
@terazus applied the changes u requested as well as
- moved data block above computed as webstorm insists. 
- moved ellipses class as it can be used generally in the main.scss
